enterprise/server/backends/redis_metrics_collector/redis_metrics_collector.go

```diff
@@ -2,17 +2,20 @@ package redis_metrics_collector
 
 import (
 	"context"
+	"flag"
 	"strconv"
 	"time"
 
 	"github.com/buildbuddy-io/buildbuddy/enterprise/server/util/redisutil"
 	"github.com/buildbuddy-io/buildbuddy/server/real_environment"
 	"github.com/go-redis/redis/v8"
+
+	_ "github.com/buildbuddy-io/buildbuddy/server/remote_cache/hit_tracker"
+	flagtypes "github.com/buildbuddy-io/buildbuddy/server/util/flagutil/types"
 )
 
-const (
-	// How long until counts expire from Redis.
-	countExpiration = 1 * 24 * time.Hour
+var (
+	countExpiration = flagtypes.Alias[time.Duration](flag.CommandLine, "cache.count_expiration")
```

Comment:
nit: the other package doesn't use the flag - might be better to move that flag here rather than using an alias

Reply:
I thought about that, but if we implement expiring stuff in other metrics collectors (like memory_metrics_collector) we'd want to have it here. I can move it if you still think we should, though.

Reply:
I thought that might have been the motivation - maybe `server/config/cache/cache.go` would be a better place then? The alias feels slightly less clean since it's matching up the two flags based on a loosely typed string

Reply:
works for me; thanks!